src/openrct2/EditorObjectSelectionSession.h

```diff
@@ -10,19 +10,21 @@
 #pragma once
 
 #include "core/EnumUtils.hpp"
+#include "core/FlagHolder.hpp"
 #include "object/Object.h"
 
 #include <vector>
 
 struct ResultWithMessage;
 
-enum EDITOR_INPUT_FLAGS
+enum class EditorInputFlag : uint8_t
 {
-    INPUT_FLAG_EDITOR_OBJECT_SELECT = (1 << 0), // Set when you want to select an object, not set when you want to deselect it.
-    INPUT_FLAG_EDITOR_OBJECT_1 = (1 << 1),
-    INPUT_FLAG_EDITOR_OBJECT_SELECT_OBJECTS_IN_SCENERY_GROUP = (1 << 2),
-    INPUT_FLAG_EDITOR_OBJECT_ALWAYS_REQUIRED = (1 << 3)
+    select, // Set when you want to select an object, not set when you want to deselect it.
+    one,
```

Comment:
It seems like this flag was simply unnamed before, correct? In that case, I think it's better to go for `unk1` instead of this.

Reply:
It was indeed unnamed. I was hoping to figure out its meaning by looking at its usage, but I’m no closer to figuring that out.